bindings_wasm_macros/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
```

Comment:
I think putting this in it's own crate was a good idea. The existing derive macros crate is getting too inflated and probably needs to be broken out into it's own pieces too.